solvers/chuffed/build.sh

```diff
@@ -1,17 +1,26 @@
 #!/bin/bash
 
-SCRIPT_DIR=$(dirname "$0")
+SCRIPT_DIR=$(realpath $(dirname "$0"))
```

Comment:
why the realpath here and not in the minion one? also same question about location of cd.

Reply:
this has already been fixed in a WIP commit - the short answer is that I am doing more stuff with minion atm.